inc/saiacl.h

```diff
@@ -952,42 +952,184 @@ typedef enum _sai_acl_table_attr_t
     SAI_ACL_TABLE_ATTR_FIELD_MACSEC_SCI,
 
     /**
-     * @brief A list of indexes into MPLS label stack (0 is top-of-the-stack)
+     * @brief Label value for MPLS label on the top
      *
      * @type bool
      * @flags CREATE_ONLY
      * @default false
      */
-    SAI_ACL_TABLE_ATTR_FIELD_MPLS_LABEL_INDEX_LIST,
+    SAI_ACL_TABLE_ATTR_FIELD_MPLS_LABEL0_LABEL,
 
     /**
-     * @brief A list of label values of MPLS labels indexed by list of indexes into MPLS label stack
+     * @brief TTL value for MPLS label on the top
      *
```

Comment:
Please add a generic label attribute for all labels
SAI_ACL_TABLE_ATTR_FIELD_MPLS_LABEL_ALL
and corresponding ttl/exp/bos attributes

Reply:
what is data type of ACL entry of SAI_ACL_TABLE_ATTR_FIELD_MPLS_LABEL_ALL? If its a list then I think problem with this has been that application is not able to provide prefixed size of matchers, it depends at "run-time" and can be arbitrarily large. This was the reason earlier changes of list were removed. Copying @lguohan  who had raised this concern before

Reply:
Discard the comment, wrong PR